### PR TITLE
Switch talking volume to segmented buttons + icons

### PR DESCRIPTION
**Pull request by jy2wong**
* Add new SegmentedControlNoDescriptions functional component
* Refactor VaccineSelector to use new functional component

### CODE REVIEW

File: src/components/calculator/ActivityRiskControls.tsx [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+  BsVolumeDownFill,
+  BsVolumeMuteFill,
+  BsVolumeUpFill,
+  BsX,
+} from 'react-icons/bs'
 import { Link } from 'react-router-dom'
 
-import { SegmentedControl } from './controls/SegmentedControl'
+import {
+  SegmentedControl,
```
Comment (beshaya) [inline src/components/calculator/ActivityRiskControls.tsx:13]: Unused import?

File: src/components/calculator/controls/SegmentedControl.tsx [hunk 2]
```diff
@@ -14,6 +14,119 @@ import { recordCalculatorOptionSelected } from 'components/Analytics'
...
+}
+
+// Mobile-friendly segmented button selector (generic version).
+// TODO(jy2wong) the old SegmentedControl still has functionality that this one
+// doesn't; port it over.
+export function SegmentedControlNoDescriptions<SourceValue>(
+  props: SegmentedProps<SourceValue>,
+): React.ReactElement {
+  const { t } = useTranslation()
+  // TODO(jy2wong) formatRiskMultiplier is copy and pasted from
```
Comment (beshaya) [inline src/components/calculator/controls/SegmentedControl.tsx:41]: You should be able to make formatRiskMultiplier a component, which would let you use hooks.

File: src/components/calculator/selectors/VaccineSelector.tsx [hunk 3]
```diff
@@ -46,103 +45,61 @@ export const VaccineSelector: React.FunctionComponent<{
...
-            >
-              <span className="segmented-wrap segmented-label">
-                {t('calculator.precautions.no_vaccine')}
-              </span>
-            </ToggleButton>
-          </ToggleButtonGroup>
-        </div>
-      </Form.Group>
-
+      <SegmentedControlNoDescriptions
```
Comment (beshaya) [inline src/components/calculator/selectors/VaccineSelector.tsx:54]: Thanks for this refactor!

**Review by beshaya — APPROVED** [review]
LGTM, with a suggestion for cleaning up formatRiskMultiplier

File: src/components/calculator/ActivityRiskControls.tsx (hunk 1 above)
Comment (beshaya) [reply src/components/calculator/ActivityRiskControls.tsx:13]: Never mind, I see it's still in use.

File: src/components/calculator/controls/SegmentedControl.tsx (hunk 2 above)
Comment (jy2wong) [reply src/components/calculator/controls/SegmentedControl.tsx]: done

**Pull request merged**